### PR TITLE
Add quick export feature and update documentation

### PR DESCRIPTION
**Pull request by fitn**

### CODE REVIEW

File: Action/QuickExportAction.php [hunk 1]
```diff
@@ -74,47 +76,60 @@ public function __construct(
...
     {
-        $response->headers->set('Content-Type', $this->options['content_type']);
-        $disposition = $response->headers->makeDisposition(
-            ResponseHeaderBag::DISPOSITION_ATTACHMENT,
-            $this->options['filename']
-        );
-        $response->headers->set('Content-Disposition', $disposition);
+        $token = $this->tokenStorage->getToken();
+        if (null === $token || !is_object($user = $token->getUser())) {
+            throw new TokenNotFoundException('You seems not authenticated anymore. Please login and try again');
```
Comment (rybus) [inline Action/QuickExportAction.php]: You are no longer authenticated. Please log in and try again.

File: Entity/Repository/CustomEntityRepository.php [hunk 2]
```diff
@@ -76,4 +77,25 @@ public function deleteFromIds(array $ids)
...
+     *
+     * @param array $referenceDataIds
+     *
+     * @return ArrayCollection
+     *
+     * @throws \InvalidArgumentException array of ids should not be empty
+     */
+    public function findByIds(array $referenceDataIds)
+    {
+        if (empty($referenceDataIds)) {
```
Comment (rybus) [inline Entity/Repository/CustomEntityRepository.php]: empty() ? isn't 0 === count() better?

File: MassEditConnector/Processor/ReferenceDataToFlatArrayProcessor.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Pim\Bundle\CustomEntityBundle\MassEditConnector\Processor;
+
+use Pim\Bundle\EnrichBundle\Connector\Processor\AbstractProcessor;
+use Pim\Component\Connector\Repository\JobConfigurationRepositoryInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ *
```
Comment (rybus) [inline MassEditConnector/Processor/ReferenceDataToFlatArrayProcessor.php]: extra line

File: MassEditConnector/Reader/ReferenceDataReader.php [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+    /**
+     * @param JobConfigurationRepositoryInterface $jobConfigRepo
+     * @param EntityManager                       $em
+     */
+    public function __construct(
+        JobConfigurationRepositoryInterface $jobConfigRepo,
+        EntityManager $em
+    ) {
+        $this->jobConfigRepo = $jobConfigRepo;
```
Comment (rybus) [inline MassEditConnector/Reader/ReferenceDataReader.php]: `$configRepository` seems better to me than cutting Repository word

File: Entity/Repository/CustomEntityRepository.php (hunk 2 above)
Comment (fitn) [inline Entity/Repository/CustomEntityRepository.php]: count needs to count so It's better to use empty

File: README.md [hunk 5]
```diff
@@ -1,20 +1,47 @@
...
         resource: "@PimCustomEntityBundle/Resources/config/routing.yml"
 
+and enable the bundle in the `app/AppKernel.php` file in the registerBundles() method:
+
+    $bundles = [
+        // ...
+        new Pim\Bundle\CustomEntityBundle\PimCustomEntityBundle(),
+    ]
+
+If you want to use the quick export and/or the mass edit features, you've to load the job fixture defined in [Resources/fixtures/jobs.yml] file and you can copy/paste its content to your installer.
```
Comment (rybus) [inline README.md]: you have [...] file and to copy/paste

File: README.md [hunk 6]
```diff
@@ -1,20 +1,47 @@
...
+
+    $ php composer.phar require akeneo/custom-entity-bundle:1.6.*
+
+Then add the following lines **at the end** of your app/config/routing.yml :
 
     pim_customentity:
         prefix: /enrich
         resource: "@PimCustomEntityBundle/Resources/config/routing.yml"
 
+and enable the bundle in the `app/AppKernel.php` file in the registerBundles() method:
```
Comment (rybus) [inline README.md]: maybe add ` to wrap registerBundles()

**Pull request closed without merging**